docs/data/charts/export/export.md

```diff
@@ -10,21 +10,19 @@ components: ScatterChartPro, BarChartPro, LineChartPro, Heatmap, FunnelChart, Ra
 
 Export is available for the following charts: `<LineChartPro />`, `<BarChartPro />`, `<ScatterChartPro />`, `PieChartPro />`, `<Heatmap />`, `<FunnelChart />` and `<RadarChartPro />`.
 
-## Print/Export as PDF
+## Overview
 
-The `apiRef` prop exposes a `exportAsPrint()` method that can be used to open the browser's print dialog.
+Charts can be exported using the browser's native print dialog or as an image.
 
-The print dialog allows you to print the chart or save it as a PDF, as well as configuring other settings.
+To enable this functionality, you need to enable the chart's toolbar by passing the `showToolbar` prop to the chart component.
 
-{{"demo": "PrintChart.js"}}
+The toolbar will then display an "Export" button that opens a menu with the export options.
```

Comment:
I think you can follow completely the Data grid structure for export

- one section to say "Just add `showToolbar` prop to see the default options"
- one section to list the different export options